### PR TITLE
Create Watson Class -> methods to check and create an environment / collection if necessary

### PR DESCRIPTION
**Pull request by horeaporutiu**

### CODE REVIEW

File: lib/watson.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
+var request = require('request');
+var jsonfile = require('jsonfile');
+
+var DiscoveryV1 = require('watson-developer-cloud/discovery/v1');
+config = jsonfile.readFileSync(__dirname + '/config.json' );
```
Comment (stevemar) [inline lib/watson.js]: you should be able to go one folder back here and find the config file in the top level directory, where they are usually located by doing:

config = jsonfile.readFileSync(__dirname + '/../config.json');

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,2 @@
 /node_modules
-config.json
\ No newline at end of file
+lib/config.json
```
Comment (stevemar) [inline .gitignore]: stuff in a lib folder is usually static, i think you have this here cause of a line in the watson.js file, let's get that looking at the top level where config related files usually live

File: app.js [hunk 3]
```diff
@@ -96,7 +85,20 @@ function scrapeEveryPage(totalPages, arrayOfReviews, productId) {
...
             console.log('Total reviews parsed: ' + totalReviews);
-            watsonAddDocument(arrayOfReviews).then(discoveryQuery);
+            watson.getDiscoveryEnvironments().then(function(result){
+              var envID = result;
+              watson.getDiscoveryConfigurations(result).then(function(output){
+                var configID = output
+                watson.getDiscoveryCollections(envID, configID).then(function(output){
+                    watson.watsonAddDocument(arrayOfReviews, output).then(function(){
+                      setTimeout(function(){
+                        console.log('about to hit setTime out from app.js')
```
Comment (stevemar) [inline app.js]: "hit setTime out" seems like a typo

**Review by stevemar — CHANGES_REQUESTED** [review]

**Comment by horeaporutiu** [comment]
committed new changes

**Review by stevemar — APPROVED** [review]

**Pull request merged**